src/pages/home/report/PureReportActionItem.tsx

```diff
@@ -241,6 +241,12 @@ type PureReportActionItemProps = {
 
     /** A message related to a report action that has been automatically forwarded */
     reportAutomaticallyForwardedMessage?: string;
+
```

Comment:
I can't comment on the line, but `ReportPreview` in line 736 is still directly connected to Onyx, which means it doesn't show correct report data, e.g.

https://github.com/user-attachments/assets/b03a319c-ddb3-422c-8ed6-3717c3cea293